content/en/account_management/billing/usage_metrics.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Estimated Usage Metrics
+kind: faq
+---
+
+## Overview
+
+Usage metrics are estimates of your current Datadog usage in near real-time. They enable you to:
+
+* Graph your estimated usage 
+* Create monitors around your estimated usage based on thresholds of your choosing
+* Get instant alerts of spikes or drops in your usage
+* Assess the potential impact of code changes on your usage in near real-time
+
+**Note:** These usage metrics are estimates that won't always match up to billable usage given their real-time nature. There is a 10-20% difference between estimated usage and billable usage on average.  
+
+{{< img src="account_management/billing/usage-metrics-01.png" alt="Dashboard Example" responsive="true">}}
+
+### Types of Usage
+
+Estimated usage metrics are generally available for the following usage types: 
+
+| Usage Type         | Metric                                                                                              |
+|--------------------|-----------------------------------------------------------------------------------------------------|
+| Infra. Hosts       | `datadog.estimated_usage.hosts`                                                                     |
+| Containers         | `datadog.estimated_usage.containers`                                                                |
+| Custom Metrics     | `datadog.estimated_usage.metrics.custom`                                                            |
+
+{{< img src="account_management/billing/usage-metrics-02.png" alt="Metric Names" responsive="true">}}
+
+More usage types will be added in the near future.
```

Comment:
```suggestion
```
We don't talk about the future in the docs

Reply:
Makes sense.